mne_bids/io.py

```diff
@@ -47,7 +47,7 @@ def _parse_ext(raw_fname, verbose=False):
 
 
 def _read_raw(raw_fname, electrode=None, hsp=None, hpi=None, config=None,
-              montage=None, verbose=None):
+              montage=None, verbose=None, allow_maxshield=None):
```

Comment:
why None? isn't this supposed to be boolean? Let's use False by default as in MNE?